frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_file_exporter_widget.dart

```diff
@@ -182,3 +209,30 @@ extension AppFlowy on dartz.Either {
     return null;
   }
 }
+
+class _AppFlowyFileExporter {
+  static Future<void> exportToPath(String path, List<ViewPB> views) async {
+    final documentService = DocumentService();
+    for (final view in views) {
+      String? content;
+      String? extension;
```

Comment:
I'd avoid using a keyword as a variable name. Maybe change `extension` to `fileExtension`?

Reply:
+ 1